pkgs/top-level/all-packages.nix

```diff
@@ -5960,7 +5960,7 @@ with pkgs;
 
   merriweather-sans = callPackage ../data/fonts/merriweather-sans { };
 
-  meson = callPackage ../development/tools/build-managers/meson {
+  meson = callPackage ../by-name/me/meson/package.nix {
```

Comment:
We might as well remove those inherits which don't aid cross and otherwise the move is kinda meh.